src/root-config-guide/root-config-guide.component.js

```diff
@@ -0,0 +1,192 @@
+import React, { useContext } from "react";
+import PageHeader from "../shared/page-header.component";
+import { useCss } from "kremling";
+import { LocalStorageContext } from "../shared/use-local-storage-data.hook";
+import { SlackLink } from "../shared/links.component";
+import Code from "../shared/code.component";
+import { sharedDepsImportMap } from "../shared/registered-app.component";
+
+export default function HtmlFile(props) {
+  const scope = useCss(css);
+  const [copying, setCopying] = React.useState(false);
+  React.useEffect(() => {
+    if (copying) {
+      const timeoutId = setTimeout(() => {
+        setCopying(false);
+      }, 500);
+
+      return () => clearTimeout(timeoutId);
+    }
+  }, [copying]);
+  const { application } = useContext(LocalStorageContext);
+  if (!application)
+    return <p>You need to register at least one application!</p>;
```

Comment:
Since the link is available in the sidebar, I just made this check so the playground won't explode when users try to explore the playground, before actually going through the steps

Reply:
Oh good point I didn't realize this was in the sidebar